tests/playground.html

```diff
@@ -584,9 +587,10 @@ <h1>Blockly Playground</h1>
   </form>
   <select id="themeChanger" name="theme" onchange="changeTheme()">
     <option value="classic">Classic</option>
-    <option value="modern">Modern</option>
```

Comment:
Why remove modern from the playground?

Reply:
You can require it separately in the playground.html file

Reply:
ok sounds good